hardware/src/base-station-sketch2/CmdMessenger.cpp

```diff
@@ -0,0 +1,680 @@
+/*
+  CmdMessenger - library that provides command based messaging
```

Comment:
These files make up an Arduino "library" , so should follow the same conventions as we use of other libraries